esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -182,9 +184,14 @@ void CWsDfuEx::init(IPropertyTree *cfg, const char *process, const char *service
 
     setDaliServixSocketCaching(true);
 
+#ifdef _CONTAINERIZED
+    IERRLOG("CONTAINERIZED(CWsDfuEx::init)");
+    maxFileAccessExpirySeconds = defaultMaxFileAccessExpirySeconds;
+#else
     factory.setown(getEnvironmentFactory(true));
     env.setown(factory->openEnvironment());
     maxFileAccessExpirySeconds = serviceTree->getPropInt("@maxFileAccessExpirySeconds", defaultMaxFileAccessExpirySeconds);
```

Comment:
this line could be outside of #ifdef (there are many other serviceTree->getXX 's outside).
And I _think_ Tony's impl. transforms converts yaml prop. trees to legacy serviceTree's so this will still work if set into the esp service yaml as a property.